survey/exporter/tex/question2tex_sankey.py

```diff
@@ -73,13 +73,26 @@ def tex(self, other_question):
                     q2.append(answer_to_q2)
         df = DataFrame(data={self.question.text: q1, other_question.text: q2})
         name = "tex/q{}_vs_q{}".format(self.question.pk, other_question.pk)
-        sankey(
-            df[self.question.text],
-            df[other_question.text],
-            aspect=20,
-            fontsize=10,
-            figureName=name,
-        )
+        if platform.system() == "Windows":
+            from pySankey.sankey import sankey
+
```

Comment:
This should not have to change between OS. Are you sure you're using the same PySankey on windows? It seems like you're using pysankey 0.0.1 on windows while this project uses pysankeybeta.

Reply:
I checked pysankey version. It is `0.0.1`.
I tried to install `pysankeybeta` but it failed.

```text
PS D:\Documents\VisualStudioCode\django-survey> pip3 install pySankeyBeta
Collecting pySankeyBeta
  Using cached https://files.pythonhosted.org/packages/d3/5d/dc33ef5c130f4b45ac362d85ec2a5bb3d2aaea32fc7e159233b96a8c6724/pySankeyBeta-1.0.4.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\username\AppData\Local\Temp\pip-install-chhyfjal\pySankeyBeta\setup.py", line 4, in <module>
        long_description = fh.read()
    UnicodeDecodeError: 'cp932' codec can't decode byte 0x97 in position 1819: illegal multibyte sequence

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\Users\username\AppData\Local\Temp\pip-install-chhyfjal\pySankeyBeta\
```

This reason guesses that [this line](https://github.com/Pierre-Sassoulas/pySankey/blob/73b9b0dc2f34624c73cee741379797784b9162f6/setup.py#L3) needs `encoding="UTF-8"`.

Reply:
I'll make PySankeyBeta compatible with windows, using your suggestion, thank you.

Reply:
I created [PR](https://github.com/Pierre-Sassoulas/pySankey/pull/2) for PySankeyBeta.
I tried to install PySankeyBeta including [its PR](https://github.com/Pierre-Sassoulas/pySankey/pull/2).
In conclusion, this logic becomes unnecessary.
I am going to back to status of forked this package.

Reply:
I released a new version of PySankeyBeta (1.0.5) and updated the master branch's setup.py. You can rebase your branch on master :)

Reply:
Thank you for merging PR to PySankeyBeta!
I merged master into this branch and checked that PySankeyBeta (1.0.5) is installed by `pip3 install -e ".[dev]"`.